timemachine/cpp/src/kernels/k_lambda_transformer_jit.cuh

```diff
@@ -54,13 +54,14 @@ void __global__ k_compute_w_coords(
 
 } // 0 or 1, how much we offset from the plane by )
 
-void __global__ k_permute_interpolated(
+void __global__ k_gather_interpolated(
     const double lambda,
     const int N,
-    const unsigned int *__restrict__ perm,
-    const double *__restrict__ d_p,
-    double *__restrict__ d_sorted_p,
-    double *__restrict__ d_sorted_dp_dl) {
+    const unsigned int *__restrict__ idxs,
+    const double *__restrict__ d_p0, // initial parameters
```

Comment:
Would be good to indicate that d_p0 and d_p1 must have the same shape

Reply:
Thanks, added in f882a97

Reply:
Its a bit confusing since P is not defined, but does the job